tests/stubtest_allowlists/py37.txt

```diff
@@ -77,3 +77,62 @@ collections.Generator.gi_running
 collections.Generator.gi_yieldfrom
 collections.Mapping.get  # Adding None to the Union messed up mypy
 collections.Sequence.index  # Supporting None in end is not mandatory
+
+# Exists at runtime, but missing from stubs
+_dummy_thread.RLock
+_imp.source_hash
+asyncio.BufferedProtocol.eof_received
+asyncio._enter_task
+asyncio._leave_task
+asyncio._register_task
+asyncio._unregister_task
+asyncio.protocols.BufferedProtocol.eof_received
+contextvars.ContextVar.__class_getitem__
+distutils.command.bdist_wininst
+distutils.command.bdist_wininst.bdist_wininst
+distutils.dist.DistributionMetadata.set_classifiers
+distutils.dist.DistributionMetadata.set_keywords
+distutils.dist.DistributionMetadata.set_platforms
+distutils.sysconfig.expand_makefile_vars
+distutils.sysconfig.get_python_version
+dummy_threading.Barrier
+dummy_threading.BoundedSemaphore
+dummy_threading.BrokenBarrierError
+dummy_threading.Condition
+dummy_threading.Event
+dummy_threading.Lock
+dummy_threading.RLock
+dummy_threading.Semaphore
+dummy_threading.TIMEOUT_MAX
+dummy_threading.Thread
+dummy_threading.ThreadError
+dummy_threading.Timer
+dummy_threading.active_count
+dummy_threading.current_thread
+dummy_threading.enumerate
+dummy_threading.get_ident
+dummy_threading.local
+dummy_threading.main_thread
+dummy_threading.setprofile
+dummy_threading.settrace
+dummy_threading.stack_size
+html.parser.HTMLParser.unescape
+importlib.util.source_hash
+lib2to3.pgen2.tokenize.COLONEQUAL
+platform.popen
+plistlib.Data.asBase64
+plistlib.Data.fromBase64
+re.Pattern.scanner
```

Comment:
Same as in the py310 entry, I'd put this in a "Won't fix" section with a ref to #6405